src/App.jsx

```diff
@@ -1,16 +1,36 @@
-import React, { useState } from "react";
-import "./App.css";
-import DogList from "./components/DogList";
+import { useState, useEffect } from 'react';
+import './App.css';
+import axios from 'axios';
+import DogList from './components/DogList';
 
-const DOGS = [
-  { id: 1, name: "flasky", age: "1", breed: "golden doodle", chip: "5388" },
-  { id: 2, name: "sparky", age: "3", breed: "golden doodle", chip: "7269" },
-  { id: 3, name: "spot", age: "10", breed: "golden doodle", chip: "" },
-];
+export const URL = 'https://ada-flasky.onrender.com/dogs';
 
 const App = () => {
   
-  const [dogs, setDogs] = useState(DOGS);
+  const [dogs, setDogs] = useState([]);
+  const [status, setStatus] = useState('Loading...');
```

Comment:
It doesn't look like this is being used for anything. I'd probably go with removing it, but we could include a comment about showing some sort of busy loading behavior.